.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "editor.codeActionsOnSave": {
-    "source.organizeImports": true,
-    "source.fixAll.eslint": true
+    "source.organizeImports": "explicit",

```

Comment:
VSCode updated these values automatically. I haven't checked but I assume this is "the new way".